Francis

francis19 opened the pull request:
I added the 'Complete' button and the 'Clear Completed Tasks' button. Feel free to remove my 'Complete' button so Leslie can do her own piece of the project.

Edit: Fixed a bug. Everything is working now.

matthewlv30 committed:
blah

matthewlv30 committed:
blah

Commit:
Merge branch 'master' into francis

Commit:
Clear Completed Tasks Button

I added the 'Complete' button and the 'Clear Completed Tasks' button.
Feel free to remove my 'Complete' button so Leslie can do her own piece
of the project.

Commit:
Fixed Some Bugs

ntopper commented:
you done goofed, merge request into our fork, this is the main repo our project is forked from. Go ahead and close this out.

Commit:
Eliminated 'size' and simplified the AngularJS. Works perfectly.

Commit:
Simplified AngularJS Code

Eliminated ‘size’ variable and made the ‘completed’ array not contain
booleans.
